lib/polymorpheus.rb

```diff
@@ -25,6 +25,7 @@ module ConnectionAdapters
   end
 
   def self.require_foreigner?
+    ActiveRecord::VERSION::MAJOR < 5 &&
     !(::ActiveRecord::VERSION::MAJOR >= 4 &&
       ::ActiveRecord::VERSION::MINOR >= 2)
```

Comment:
Can this be simplified?

`ActiveRecord::VERSION::MAJOR == 4 && ActiveRecord::VERSION::MINOR < 2`